src/test/java/org/fsg1/fmms/backend/resources/CurriculumResourceTest.java

```diff
@@ -0,0 +1,178 @@
+package org.fsg1.fmms.backend.resources;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.filter.log.RequestLoggingFilter;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import org.glassfish.jersey.internal.inject.AbstractBinder;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.test.JerseyTest;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.fsg1.fmms.backend.services.CurriculumService;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+import static io.restassured.RestAssured.get;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.iterableWithSize;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.*;
+
+public class CurriculumResourceTest extends JerseyTest {
+
+    private static RequestSpecification spec;
+    private ObjectMapper mapper = new ObjectMapper();
+
+    @Mock
+    private CurriculumService service;
+
+    @BeforeClass
+    public static void initSpec() {
+        spec = new RequestSpecBuilder()
+                .setContentType(ContentType.JSON)
+                .setBaseUri("http://localhost:9998/")
```

Comment:
Please use a higher port, some above 30k

Reply:
This port is used by the JerseyTest Grizzly container when testing. Is it useful to change it for tests?

Reply:
If the port is given by the framework, then leave it to the convention. If it is chosen by you and the actual value does not matter, then change it to something high.